packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -704,6 +704,25 @@ const schema: SchemaType<DbComment> = {
     hidden: true,
     ...schemaDefaultValue(false),
   },
+  
+  // How well does ModGPT (GPT-4) think this comment adheres to forum norms and rules? (currently EAF only)
+  modGPTAnalysis: {
+    type: String,
+    optional: true,
+    canRead: ['sunshineRegiment', 'admins'],
+    canCreate: ['admins'],
+    canUpdate: ['admins'],
+    hidden: true
+  },
+  // This should be one of: Intervene, Consider reviewing, Don't intervene
+  modGPTRecommendation: {
+    type: String,
```

Comment:
This can also have an `allowedValues` field which accepts a tuple of string values, and then the code-gen will correctly generate the narrower union type instead of just `string`.  (You can look at `moderationTemplates/schema` for an example.)

Reply:
Also I think both new fields should be `nullable: true`, unless you want to backfill all the old comments

Reply:
Oh, sorry, I wrote that without checking that we'd be constraining whatever we put into this field to be one of those values.